server/server.js

```diff
@@ -45,14 +45,25 @@ app.post('/api',
     res.status(200).json(res.locals.boards)
   })
 
+app.get('/boards/:board_id', 
+  // sessionController.isLoggedIn,
+  // userController.getBoardIds, 
+  boardController.getBoard, 
+  (req, res) => {
+    res.status(200).json(res.locals.board)
+})
+
 app.post(
   "/login",
   userController.verifyUser,
   sessionController.startSession,
   cookieController.setSSIDCookie,
   (_, res) => {
     console.log("completing post request to '/login");
-    res.sendStatus(200);
+    // res.redirect('/secret');
+    // res.sendStatus(200);
+    res.status(200).json(res.locals.user);
```

Comment:
From the way you setup `client/pages/LoginPage.jsx`, it looks like you really only need board ID, correct? With the implementation of `express-session` in https://github.com/CS56-Danger-Noodle/Scratch-Project/pull/5, we have the ability to check the username with session without you passing the username through (I believe). In that case, I can just send back `board_id` without the need to send username (redundant) or password (sensitive info and should not be sent back to front-end anyway).

You don't need to change this right now, though after my PR is merged we may want to go back and fix this in the front end when you refactor that code

Reply:
I see what you mean. I think you are right, we should not be passing the password for sure!! And username is redundant. We discussed those details briefly but figured it would be good to confirm with you. Yes, the only thing we really need passed to frontend is the board_id array. For now, we are only directly sending the user to the very first board in their array because we have yet to implement the 'boards feed' page.

Reply:
Ya for this, I'd say leave this for now and we can change this later

Reply:
updated server response to include ONLY board_ids array and user_id